test/controllers/kwargs_test.rb

```diff
@@ -601,8 +593,8 @@ def setup
     end
 
     define_method("test_xhr_#{verb}_new_params_and_session_and_format") do
-      send(verb.to_sym, :test_kwargs, format: :json, params: { session: { a: 1 } }, xhr: true)
-      assert_equal({ 'session' => { 'a' => 1 }, 'format' => 'json' }, assigns(:params))
+      send(verb.to_sym, :test_kwargs, format: :json, params: { session: { a: "foo" } }, xhr: true)
```

Comment:
There is a change in Rails 5 that causes this assertion to fail (it's a string `"1"` instead of `1`). Rather than introduce a conditional for that, changed the value to a string.